cmd/edgectl/aes_install.go

```diff
@@ -709,20 +715,29 @@ func (i *Installer) Perform(kcontext string) error {
 		i.Report("dns_name_failure", ScoutMeta{"code", resp.StatusCode}, ScoutMeta{"err", message})
 		i.show.Println("-> Failed to create a DNS name:", message)
 		i.show.Println()
-		i.ShowWrapped(noTlsSuccess)
+		i.ShowWrapped(color.Bold.Sprintf(noTlsSuccess))
+		i.show.Println()
+		
 		i.ShowWrapped("If this IP address is reachable from here, you can access your installation without a DNS name.")
-		i.ShowWrapped(fmt.Sprintf(loginViaIP, i.address))
+		i.ShowWrapped(loginViaIP)
+		i.show.Println(color.Bold.Sprintf("$ edgectl login -n ambassador %s", i.address))
+
+		i.show.Println()
 		i.ShowWrapped(loginViaPortForward)
+		i.show.Println(color.Bold.Sprintf("$ kubectl -n ambassador port-forward deploy/ambassador 8443 &"))
+		i.show.Println(color.Bold.Sprintf("$ edgectl login -n ambassador 127.0.0.1:8443"))
+		i.show.Println()
+
 		i.ShowWrapped(seeDocs)
 		return nil
 	}
 
-	i.hostname = string(content)
-	i.show.Println("-> Acquiring DNS name", color.Bold.Sprintf(i.hostname))
-
 	// Wait for DNS to propagate. This tries to avoid waiting for a ten
 	// minute error backoff if the ACME registration races ahead of the DNS
 	// name appearing for LetsEncrypt.
+	i.hostname = string(content)
+	i.show.Println("-> Acquiring DNS name", color.Bold.Sprintf(i.hostname))
+
 	if err := i.loopUntil("DNS propagation to this host", i.CheckHostnameFound, lc2); err != nil {
```

Comment:
I don't agree with this change. The comment applies to the `loopUntil` call, but you have moved it far away.

Reply:
I'll move it to the loopUntil block, makes sense.